README.md

```diff
@@ -14,33 +14,33 @@ At the time of writing the current release is `1.3`
 ## Installing Dependencies
 
 This project uses Docker and Docker Compose. For Ubuntu 16.04, an installation
-script `install-docker.sh` is provided for convenience.
+script `install-docker.sh`, is provided for convenience.
```

Comment:
There should be a comma before `install-docker.sh` as well (or no commas at all) 😄